tests/unit/amo/components/TestRatingManager.js

```diff
@@ -106,10 +110,32 @@ describe(__filename, () => {
       apiState: signedInApiState,
       userId,
       addonId: addon.id,
+      addonSlug: addon.slug,
       versionId: version.id,
     });
   });
 
+  it('does not load a saved review when none exists', () => {
+    const userId = 12889;
+    const addon = createInternalAddon({ ...fakeAddon, id: 3344 });
+    const version = {
+      ...fakeAddon.current_version,
+      id: 9966,
+    };
+    const loadSavedReview = sinon.spy();
+
+    render({
+      apiState: signedInApiState,
```

Comment:
Too bad this one does not come from `dispatchSignInActions().store`.

Reply:
This whole test file needs to be rewritten. Yes, it should be using `dispatchSignInActions()`.